### DISCUSSION TITLE
Splitter in Row Design

### DISCUSSION THREAD

**Discussion by NeoCoderMatrix86** [Megabit/Blazorise#1796] [Q&A]
Hello,
I'm using Blazorise inside a little project (https://neocodermatrix86.github.io/AudioCuesheetEditor/).
There I have on desktop a Row layout:
```
<Row>
  <Column>[...]</Column>
  <Column ColumnSize="ColumnSize.IsQuarter.OnDesktop.IsFull.OnTablet">[...]</Column>
</Row>
```

Is it possible to have something like a vertical splitter, in order to allow the breakpoint set by user? Currently the breakpoint is set hard to "IsQuarter", but what if the user has a really wide display and does want to have its own design?!
Thanks for your help :)!

**Comment by stsrki** [comment]
I was experimenting with the Splitter component https://github.com/stsrki/Blazorise/pull/1366.

But it's still not ready.

**Reply by NeoCoderMatrix86** [reply]
No problem, I have currently parked the issue and wait for version 0.9.3 to come up :).

**Reply by NeoCoderMatrix86** [reply]
I just wanted to ask, what is the current state of the Splitter thing? Is it already contained in version 1.0.2?

**Comment by stsrki** [comment]
It was abandoned some time ago. We had problems making it a general-purpose component. At least with the approach we wanted. I think we can consider making it again but it is not a top priority at the moment.

**Reply by NeoCoderMatrix86** [reply]
Ok, thanks for your answer, I will wait for a release :)
